index.html

```diff
@@ -179,21 +223,18 @@ <h2 class="title"> Multi-Post
 
       <!-- About Myself Section -->
 
-      <Section class="about-myself">
-        <div class="header-myself">
-          <h2 class="heading-about">About <br> Myself</h2>
-        </div>
-
-        <div class="paragraph3">
+      <Section id="about-myself">
+        <div class="about-about-myself">
+          <div class="header-myself">
```

Comment:
Kindly consider the about me section and modify your design to match your chosen design on Figma. It will look better that way. Check the comment above for more reason you should follow a design when given.

Your design:
![image](https://user-images.githubusercontent.com/20554583/186801503-1ba14ce4-1a0f-4d2f-a6de-e5d66f051947.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/186801556-37a47d5a-c74a-46dd-9a7b-a822988c5918.png)

Take note of the following:

- The skills section takes up more space than the about info section.
- The arrow following each skill is centred vertically
- Framworks is correctly spelt as Frameworks